source/dyaml/node.d

```diff
@@ -2446,12 +2428,30 @@ struct Node
         // Safe wrapper for getting a value out of the variant.
         inout(T) getValue(T)() @trusted inout
         {
-            return value_.get!T;
```

Comment:
can we make this method `@safe`? if not, we should probably document why it's `@trusted`

Reply:
This one was switched to `@safe`.